CODE_OF_CONDUCT.md

```diff
@@ -59,12 +59,32 @@ reported by contacting the project team at jashkenas@gmail.com. The project team
 will review and investigate all complaints, and will respond in a way that it deems
 appropriate to the circumstances. The project team is obligated to maintain
 confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
-
 Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
 members of the project's leadership.
 
+## Moderation
+
+Edits of another user's comment must be clearly marked with "**edit**", the
+moderator's username, and a timestamp for each occurrence. The only acceptable
+reasons for editing another user's comment are:
+
+1. to edit out violations of this Code of Conduct. These edits must include a
+   rationale.
+2. to direct future readers to a relevant point later in the conversation
+   (usually the resolution). These edits must be append-only.
```

Comment:
@megawac This section discusses editing of others' comments. You are free to edit your own PR comment before anyone has responded, or via an append-only change once someone else has responded. You are also free to leave a comment attributed to your own name that says `fixes #123`. I don't think it would be right to silently alter someone's speech, no matter how small you feel it is. Notice that I've not given an exception for correcting someone else's typos/grammar/formatting because that would impose on their freedom of expression (answering @akre54's question).